lib/stdlib/src/io_lib.erl

```diff
@@ -449,6 +449,16 @@ write_binary_body(B, D, T, Acc) when D =:= 1; T =:= 0->
     {["..."|Acc], B};
 write_binary_body(<<X:8>>, _D, _T, Acc) ->
     {[integer_to_list(X)|Acc], <<>>};
+write_binary_body(<<0:24,Rest0/bitstring>>, D, T, Acc) ->
```

Comment:
It would probably be worth mentioning that this will fire only if there are 3 or more zeroed bytes.

Reply:
Right. It's because `0:16,` is longer than `0,0,`. But it's an arbitrary decision nevertheless.